README.md

```diff
@@ -198,10 +199,10 @@ So far we've only been dealing with the standard output of a command, but what i
 ```wdl
 task hello {
   command {
-    echo 'hello ${name}!'
+    echo 'hello ${name}!' > test.out
   }
   output {
-    String response = read_string(stdout())
+    String response = read_string("test.out")
```

Comment:
I wasn't sure, but thought maybe that was the intent of the section. It looked like the example under [modifying-task-outputs](https://github.com/broadinstitute/wdl/blob/master/README.md#modifying-task-outputs) was precisely the same as [referencing-files-on-disk](https://github.com/broadinstitute/wdl/blob/master/README.md#referencing-files-on-disk), so I thought maybe there was a copy-paster error and a change like that was intended to demonstrate referencing files on disk.